src/main/scala/cognite/spark/v1/udf/CogniteUdfs.scala

```diff
@@ -0,0 +1,143 @@
+package cognite.spark.v1.udf
+
+import cats.effect.IO
+import cognite.spark.v1.Constants.{DefaultBaseUrl, DefaultMaxRetries}
+import cognite.spark.v1.{CdfSparkException, CdpConnector, Constants}
+import com.cognite.sdk.scala.common.ApiKeyAuth
+import com.cognite.sdk.scala.v1.{FunctionCall, GenericClient}
+import com.softwaremill.sttp.SttpBackend
+import io.circe.{Json, JsonObject, parser}
+import org.apache.spark.sql.SparkSession
+
+class CogniteUdfs(sparkSession: SparkSession) extends Serializable {
+
```

Comment:
We can do `implicit val backend: SttpBackend[IO, Nothing] = CdpConnector.retryingSttpBackend(maxRetries)` just once here, I think? Unless we need to get a new one for each call.